plugins/grpc/stats/client_handler.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package stats
+
+import (
+	"golang.org/x/net/context"
```

Comment:
[`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) would complain about the order of the imported packages.

We do those checks on travis: https://github.com/menghanl/grpc-go/blob/master/vet.sh#L51-L54

Reply:
Ok. Fixed.